argon2/include/blake2b.h

```diff
@@ -0,0 +1,94 @@
+#ifdef __cplusplus
```

Comment:
Why do we need another blake implementation? You could reuse the ones we have (that's why we did that first ;))